pointerlock/interfaces.window.js

```diff
@@ -5,22 +5,14 @@
 
 // https://w3c.github.io/pointerlock/
 
-promise_test(async () => {
-  const html = await fetch('/interfaces/html.idl').then(r => r.text());
-  const dom = await fetch('/interfaces/dom.idl').then(r => r.text());
-  const uievents = await fetch('/interfaces/uievents.idl').then(r => r.text());
-  const idl = await fetch('/interfaces/pointerlock.idl').then(r => r.text());
-
-  const idl_array = new IdlArray();
-  idl_array.add_idls(idl);
-  idl_array.add_dependency_idls(uievents);
-  idl_array.add_dependency_idls(dom);
-  idl_array.add_dependency_idls(html);
-
-  idl_array.add_objects({
-    Document: ["window.document"],
-    Element: ["window.document.documentElement"],
-    MouseEvent: ["new MouseEvent('foo')"]
-  });
-  idl_array.test();
-}, 'pointerlock interfaces.');
+idl_test(
+  ['pointerlock'],
+  ['uievents', 'html', 'dom'],
```

Comment:
The dependency idls are added in a different order here than in the previous revision. Does this make a difference?

Reply:
Actually, if you look at https://github.com/web-platform-tests/wpt/pull/11833/files/e3f9f7b704f6f50514f1db4dfaf16747cb5f7c3d#diff-0375a5322ffb1328cfc6a75c965c8ea2L16 `uievents` was first. I double-checked and all the tests are happily passing with this new order.

Reply:
uievents is still first but dom and html are in reverse order, although if all tests pass in the new order should be good